src/commands/revenue.js

```diff
@@ -35,6 +37,13 @@ function definePeriods(args, options) {
     periods.push([start, end]);
   }
 
+  function createYearPeriod(year) {
+    const start = moment(year, 'YYYY').utc();
+    // If year is the current year the end should be today, otherwise it should be the end of the year.
+    const end = year === currentYear ? defaultEndTime : moment(year, 'YYYY').utc().endOf('year');
```

Comment:
Here you should remove `.utc()`. Otherwise the server will respond with the users year, which is different from the UTC year (the revenue year).